packages/thumbprint-react/components/HorizontalRule/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styles from './index.module.scss';
+
+const propTypes = {
+    /**
+     * Renders the horizontal rule with the thumbtack styles.
+     */
+    lineStyle: PropTypes.oneOf(['original', 'dotted', 'dashed']),
```

Comment:
To keep the options consistent can 'original' be changed to 'solid'?

Reply:
Agreed here.